backend/src/openbeheer/utils/gherkin_e2e.py

```diff
@@ -251,14 +251,12 @@ def user_selects_tab(self, page: Page, name: str = "") -> None:
             button = page.get_by_role("tab", **kwargs)
             button.click()
             page.wait_for_load_state("networkidle")
+            button = page.get_by_role("button", exact=True, **kwargs)
+            button.click()
 
         # Actions
 
-        def user_clicks_on_button(self, page: Page, name: str = "") -> None:
-            kwargs = {}
-            if name:
-                kwargs.update({"name": name})
-
+        def user_clicks_on_button(self, page: Page, **kwargs) -> None:
```

Comment:
:thought_balloon: `page.get_by_role` has good and precise doc string and annotations. This abstraction severs me from that.
I don't see an easy solution, but when I squint this is only a partial application and a function composition.